apple/macbook-pro/12-1/default.nix

```
@@ -16,7 +16,10 @@
     # https://bugzilla.kernel.org/show_bug.cgi?id=101681#c116.
     # Also brcmfmac could randomly crash on resume from sleep.
     powerUpCommands = lib.mkBefore "${pkgs.kmod}/bin/modprobe brcmfmac";
-    powerDownCommands = lib.mkBefore "${pkgs.kmod}/bin/rmmod brcmfmac";
+    powerDownCommands = lib.mkBefore ''
+      ${pkgs.kmod}/bin/lsmod | ${pkgs.gnugrep}/bin/grep -q "^brcmfmac_wcc" && ${pkgs.kmod}/bin/rmmod -f -v brcmfmac_wcc
```

Can we not just unconditionally do this:

```suggestion
      ${pkgs.kmod}/bin/rmmod -f -v brcmfmac_wcc 2>/dev/null
```